### DISCUSSION TITLE
Saving data to RDSS

### DISCUSSION THREAD

**Discussion by Jina-Kim-1108** [l3atbc-datadog/pushkin#113] [General]
Hi!
I am planning to save the participants' data in rdss (research data storage service) provided by the university. And I am struggling how to write a code for it. Could you give me some tips?

**Comment by jkhartshorne** [comment]
@Jina-Kim-1108 Can you please elaborate? I'm not sure this is a Pushkin question?

**Reply by jkhartshorne** [reply]
@Jina-Kim-1108 just following up on this. Do you still have a question? If so, can you please provide more information?
